lib/internal/crypto/hash.js

```diff
@@ -58,11 +58,24 @@ const {
   isArrayBufferView,
 } = require('internal/util/types');
 
+const { getOptionValue } = require('internal/options');
+
 const LazyTransform = require('internal/streams/lazy_transform');
 
 const kState = Symbol('kState');
 const kFinalized = Symbol('kFinalized');
 
+/**
+ * @param {string} name
+ */
+function normalizeAlgorithm(name) {
+  return name.toLowerCase().replace('-', '');
+}
+
+const pendingDeprecation = getOptionValue('--pending-deprecation');
```

Comment:
Can you use `pendingDeprecate` from `internal/util` instead?

Reply:
`pendingDeprecate` from `internal/util` is for wrapping functions, that doesn't apply here. Would you make a suggestions on how you'd like to see `pendingDeprecate` used?